ZupIT/ritchie-cli
# [FEATURE] Supporting password fields in formula config

**- What I did**
Supporting password fields in formula config

**- How I did it**
Adding a new type `password` in inputs handler to call the appropriate prompt

**- How to verify it**
Add a password field in any formula config.json
run the formula and you will see `*` when you type some value

**- Description for the changelog**

Supporting password fields in formula config
